core/capabilities/webapi/webapicap/webapi_target-schema.json

```diff
@@ -0,0 +1,62 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://github.com/smartcontractkit/chainlink/v2/core/capabilities/webapi/webapicap/web-api-target@1.0.0",
+    "description": "A target that sends HTTP requests to a URL",
+    "$defs": {
+        "TargetPayload": {
+            "type": "object",
+            "properties": {
+                "url": {
+                    "type": "string",
+                    "description": "The URL to send the request to"
+                },
+                "method": {
+                    "type": "string",
+                    "description": "The HTTP method to use for the request"
+                },
+                "headers": {
+                    "type": "object",
+                    "description": "The headers to include in the request",
+                    "additionalProperties" : {
+                        "type": "string"
+                    }
+                },
+                "body": {
+                    "type": "string",
+                    "description": "The body of the request"
+                }
+            },
+            "required": ["url"],
+            "additionalProperties": false
+        },
+        "TargetConfig": {
+            "type": "object",
+            "properties": {
+                "timeoutMs": {
+                    "type": "integer",
+                    "description": "The timeout in milliseconds for the request"
+                },
+                "retryCount": {
+                    "type": "integer",
+                    "description": "The number of times to retry the request"
+                },
```

Comment:
It seems odd that we're letting users specify this themselves.  I'm assuming that 0 means to use the default, a documentation line for that would be good.  Also, maybe set the min value to 0 and the max values to something reasonable so users can't just set whatever they want.

Reply:
IMO, customers know the availability/latency of their own HTTP target endpoint best so I thought it would make sense for them to specify this. (e.g. if their availability is 99.999999%, they would specify low `retryCount`). I will update the description with the defaults and min/max.